validator/graffiti/parse_graffiti.go

```diff
@@ -0,0 +1,26 @@
+package graffiti
+
+import (
+	"io/ioutil"
+
+	"gopkg.in/yaml.v2"
+)
+
+type Graffiti struct {
+	Default  string            `yaml:"default,omitempty"`
+	Random   []string          `yaml:"random,omitempty"`
+	Specific map[uint64]string `yaml:"specific,omitempty"`
```

Comment:
In your design, you call this "validators". Why is it "specific" here? I

Reply:
I changed it to specific in the design doc but i forgot to update the example sections.

You mentioned specific in our chat and I liked it better than validators

Reply:
Sounds good. No preference here, just wanting consistency. thanks!